desc/compute/_stability.py

```diff
@@ -232,3 +232,388 @@ def _magnetic_well(params, transforms, profiles, data, **kwargs):
         0,  # coefficient of limit is V_r / V_rr, rest is finite
     )
     return data
+
+
+@register_compute_fun(
+    name="ideal ball gamma2",
+    label="\\gamma_{\\mathrm{ballooning}}=-\\omega^2",
+    units="~",
```

Comment:
As coded, this has units of $(T/m)^2$.

Reply:
Yeah, it's a scaling of (B_N/a_N)^2. It just scales up the value of lambda and as a consequence makes it dimensional.

Reply:
There are usually other constants such as mu_0 and n_0 on the right hand side that will effectively make the units [B_N^2/(mu_0 n_0) * 1/a_N^2] = [(1/s)^2]

Reply:
This has been corrected and should be truly dimensionless now.